packages/itwinui-react/src/core/Tile/Tile.tsx

```diff
@@ -184,6 +197,8 @@ export const Tile = (props: TileProps) => {
       tabIndex={isActionable && !isDisabled ? 0 : undefined}
       {...rest}
     >
+      {variant !== 'folder' ? tileName : null}
```

Comment:
nit:

```jsx
{variant !== 'folder' && tileName}
```

Reply:
this was intentional because ternary is always clearer and there is no chance of accidentally rendering `false` as a string